lib/Mojolicious/Plugin/Yancy.pm

```diff
@@ -391,6 +391,23 @@ executed in that order. E.g.:
         },
     }
 
+You can also configure filters on OpenAPI operations' outputs, this time
```

Comment:
This sounds like it's useful, but it also sounds like it could hide problems better solved by other mechanisms (such as making sure the data is formatted correctly inside the database, or making the editor handle the formats better). What kind of use-cases are you seeing for this feature?

Reply:
Specifically in the "realworld" API, they annoyingly have one- or two-level "objects" that wrap a simple input or output. Output filtering preserves the simplicity of the DB, while conforming to that shape. Since filters can do just about anything, this is only one example! Another might be to add a SHA.

Reply:
Hmm... or a "session" could start by submitting a `POST` request to a `/session` API with a token and then, like you mention, the `x-filter-output` could add the ephemeral stuff. Or data from another source (LDAP, cache, etc...). Okay.

I was thinking that you may not know I was planning on having backend hooks (hooks which could then be added by plugins): One for before/after the five methods. This may result in some things being better added to certain layers:

* Filters operate on the controller layer for the web
* Hooks operate on the backend layer for anything

Reply:
I've been thinking along vaguely similar lines in that for instance it will be good to have a hookable OpenAPI-inference phase. It may be worth having a more generalised mechanism, possibly tied to Mojolicious's "events".